src/components/AudioDetails/VAudioDetails.vue

```diff
@@ -1,12 +1,24 @@
 <template>
   <!--eslint-disable @intlify/vue-i18n/no-raw-text -->
```

Comment:
Great catch with the untranslated string! I think we can remove the eslint-disable now?